rules_poetry/poetry.bzl

```diff
@@ -21,12 +30,18 @@ def _get_python_interpreter_attr(rctx):
 def _resolve_python_interpreter(rctx):
     python_interpreter = _get_python_interpreter_attr(rctx)
 
-    if rctx.attr.python_interpreter_target:
+    if "win" in rctx.os.name:
+        interpreter_target = rctx.attr.python_interpreter_target_win
+    elif "mac" in rctx.os.name:
+        interpreter_target = rctx.attr.python_interpreter_target_mac
+    else:
```

Comment:
Can you add a comment about which os we're getting as a fallback? e.g.: `. # rctx.os.name contains 'unix'`?

Reply:
added some comments, in the fallback case we are not expecting certain OS, its really just about the location of the python binary (which, yes, is somewhat correlated to OS) [7aa2a09](https://github.com/mongodb-forks/rules_poetry/pull/1/commits/7aa2a09847cc9482b937f4381021a9cb45946821)